### PR TITLE
Update: Prepare for 1.10.0-RC1

### PR DESCRIPTION
**Pull request by LordAro**
"Scheduled" for 12:00 UTC tomorrow :)

### CODE REVIEW

File: changelog.txt [hunk 1]
```diff
@@ -1,3 +1,49 @@
...
+Change: Move autorenew setting to basic category (#7984)
+Change: Improved algorithm for transfer feeder payments (#7935)
+Change: Show volume sliders with wedges instead of boxy slider (#7902)
+Change: Auto-restart loads the original resources (e.g. save or scenario) again (#7688)
+Change: Improve readability of integer lists saved to config files (#7396)
+Fix #7976: Crash when attempting to kick the host via rcon (#7985)
+Fix #7592: Road vehicles no longer balanced between multiple road stop stations (#7979)
+Fix: Station rating effects affecting too large area (#7977)
+Fix #7974: Crash when Ctrl+click to show a collapsed vehicle group (#7975)
+Fix #7969: Crash when executing a recurive console alias (#7973)
```
Comment (glx22) [inline changelog.txt]: recursive

File: changelog.txt [hunk 2]
```diff
@@ -1,3 +1,49 @@
...
+Fix: Avoid a crash by properly resetting timetable duration when loading old savegames (#7894)
+Fix: Possible crash when post road-works cleanup removes all road pieces (#7903)
+Fix #7891: Fix crash when loading save from 1.7.2 (#7892)
+Fix #7887: Missing sound effects for some main toolbar buttons (#7888)
+Fix #6667: Avoid confusion by also recalculating bridge costs for 'spectated' AI companies (#7884)
+Fix: Allow old NewGRF industries to hide in/out cargo slots (#7882)
+Fix: [Windows] Fix bootstrap GUI with Uniscribe but no Freetype (#7878)
+Fix: Missing keycodes for hotkeys.cfg (#7850)
+Fix #7625: Ensure road infrastructure cost is correctly updated when upgrading your own roads (#7628)
+Fix #7088: Close AI/GS textfile windows when their data becomes invalid (#7094)
```
Comment (James103) [inline changelog.txt]: Could be "Close AI/GS textfile windows when their data **become** invalid"

Comment (LordAro) [reply changelog.txt]: nope, "data" is plural

File: changelog.txt [hunk 3]
```diff
@@ -1,3 +1,49 @@
...
+Change: Auto-restart loads the original resources (e.g. save or scenario) again (#7688)
+Change: Improve readability of integer lists saved to config files (#7396)
+Fix #7976: Crash when attempting to kick the host via rcon (#7985)
+Fix #7592: Road vehicles no longer balanced between multiple road stop stations (#7979)
+Fix: Station rating effects affecting too large area (#7977)
+Fix #7974: Crash when Ctrl+click to show a collapsed vehicle group (#7975)
+Fix #7969: Crash when executing a recurive console alias (#7973)
+Fix #6566: Very long loading of the maximum "zoom out" level at high resolutions (#7968)
+Fix #7952: Crash when switching input languages (#7953)
+Fix #7927: Incorrect resolution shown in "Huge screenshot" query window (#7949)
```
Comment (frosch123) [inline changelog.txt]: #7949 is part of #7550, isn't it?

File: changelog.txt (hunk 2 above)
Comment (LordAro) [reply changelog.txt]: So you are technically correct actually, but I don't like how it reads. So I just rewrote the message to avoid "data" entirely :)

**Comment by nielsmh** [comment]
Looks good to me

**Review by glx22 — DISMISSED** [review]

**Review by glx22 — APPROVED** [review]

**Pull request merged**